message-streams/javascript/.gitignore

```diff
@@ -1,4 +1,7 @@
-dist/
+dist/*
```

Comment:
Rather than adding `dist/cjs/package.json` to git, can we use the approach used here?

https://github.com/cucumber/cucumber-expressions/blob/0875a402cc7efa2d01c1031c96cd2cc448f94b78/javascript/package.json#L21

Reply:
Yeah, I saw what you did with cucumber-expressions.
Will do 👌